docs/rules/numeric-separators-style.md

```diff
@@ -0,0 +1,70 @@
+# Enforce the style of numeric separators by correctly grouping digits
+
+Enforces a convention of grouping digits thanks to the [numeric separator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Numeric_separators).
+
+This rule is fixable.
+
+
+## Fail
+
+```js
+const foo = 1_23_4444;
+const foo = 1_234.56789;
+const foo = 0xAB_C_D_EF;
+const foo = 0b10_00_1111;
+const foo = 0o1_0_44_21;
+const foo = 1_294_28771_2n;
+```
+
+
+## Pass
+
+```js
+const foo = 1_234_444;
+const foo = 1_234.567_89;
+const foo = 0xAB_CD_EF;
+const foo = 0b1000_1111;
+const foo = 0o10_4421;
+const foo = 1_294_287_712n;
+```
+
+
+## Options
+
+If you want a custom group size for a specific number type, you can specify it here.
+
+There are 5 options, `hexadecimal`, `binary`, `octals`, `bigint`, `number`, which all
+refer to their corresponging type. Each of them has to be associated with an array
+containing 2 values:
+- The first one is the minimum threshold. You can't use numeric separators if the number has
```

Comment:
How about `minimumDigits`

Reply:
It is indeed more explicit and easier to understand. Should I also update for the configuration, or only in the description, in the documentation?

Reply:
Ah, didn't notice it's array.

Reply:
Oh you're right, i forgot to change the doc after sindresorhus suggested to make it an object. I'll change that.